contentcuration/contentcuration/management/commands/exportchannel.py

```diff
@@ -188,14 +193,17 @@ def create_bare_contentnode(ccnode, default_language):
             'kind': ccnode.kind.kind,
             'title': ccnode.title,
             'content_id': ccnode.content_id,
+            'channel_id': channel_id,
```

Comment:
So we associate nodes with a channel id now? Just curious

Reply:
Yeah, we use it as an index to allow for quicker lookup.